kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/RuntimeDataServiceIntegrationTest.java

```diff
@@ -720,7 +720,7 @@ public void testGetNodeInstances() throws Exception {
             nodeInstance = queryClient.findNodeInstanceByWorkItemId(processInstanceId, nodeInstance.getWorkItemId());
             assertNotNull(nodeInstance);
```

Comment:
I think this block of asserts could be refactored into a single method - to avoid duplication.